bi_sql_editor/views/view_bi_sql_view.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+@author Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+
+    <record id="view_bi_sql_view_tree" model="ir.ui.view">
+        <field name="model">bi.sql.view</field>
+        <field name="arch" type="xml">
+            <tree colors="blue:state=='draft'; brown:state in ('sql_valid', 'model_valid')">
```

Comment:
Please replace the `colors=` syntax with the new `decoration-*` way to define colors of the rows. I think the `colors=` way is not even working anymore.